terraform/dynamo.tf.json.template.py

```diff
@@ -77,6 +77,22 @@
                                 "type": "S"
                             }
                         ]
+                    },
+                    "failed_messages_table": {
+                        "name": config.dynamo_failure_message_table_name,
```

Comment:
```suggestion
                        "name": config.dynamo_failures_table_name,
```

Reply:
```suggestion
                        "name": config.failures_table_name,
```

Unless there is precedent for the `dynamo` prefix.

Reply:
Changing variable name to `dynamo_failures_table_name` to match other dynamo related configs `dynamo_cart_table_name`, `dynamo_cart_item_table_name`, `dynamo_user_table_name`, and `dynamo_object_version_table_name`.